src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1256,10 +1256,12 @@ internal static bool GetUnificationUseSiteDiagnosticRecursive(ref DiagnosticInfo
 
         #endregion
 
+#pragma warning disable CS0419 // Ambiguous reference in cref attribute
         /// <summary>
         /// True if this symbol has been marked with the <see cref="ObsoleteAttribute"/> attribute. 
         /// This property returns <see cref="ThreeState.Unknown"/> if the <see cref="ObsoleteAttribute"/> attribute hasn't been cracked yet.
         /// </summary>
+#pragma warning restore CS0419
```

Comment:
Needed to add suppressions around `<see cref="ObsoleteAttribute"/>` because `cref` considers even inaccessible types (the custom `ObsoleteAttribute` is not accessible in this project).

Reply:
> What happens in the resulting xml docs? Which symbol does it end up binding to?

It just refers to `System.ObsoleteAttribute`, I don't think it's possible to tell which one:

```xml
<member name="P:Microsoft.CodeAnalysis.CSharp.Symbol.ObsoleteState">
  <summary>
    True if this symbol has been marked with the
    <see cref="T:System.ObsoleteAttribute"/>
    attribute. This property returns
    <see cref="F:Microsoft.CodeAnalysis.ThreeState.Unknown"/>
    if the
    <see cref="T:System.ObsoleteAttribute"/>
    attribute hasn't been cracked yet.
  </summary>
</member>
```

> Is there a way we could update the cref to use unambiguous syntax instead?

I tried but haven't found a way.